galadriel/case/state.py

```
@@ -305,6 +365,7 @@ def handle_submit(self, form_data):
         updated_data = {**form_data}
         result = self.save_case_edits(case_id, updated_data)
         if result is None: return rx.toast.error("name cannot be empty")
```

Ruff (0.15.6)

[error] Multiple statements on one line (colon)

(E701)